Whiteboard messages documentation reworked (#3570)
WeblateOrg/weblate: docs/admin/whiteboard.rst

Whiteboard messages
===================

> Provide infor to your translators.by posting messages, site-wide, per component or language.

Author: ```suggestion
Provide info to your translators.by posting messages, site-wide, per component or language.
```

> Provide infor to your translators.by posting messages, site-wide, per component or language.

Author: ```suggestion
Provide info to your translators.by posting messages, site-wide, per component, or language.
```

> Announce the purpose, deadlines, status, or specify targets for translations.

Author: ```suggestion
Announce the purpose, deadlines, status, or specify targets for translation.
```

> Adding whiteboard messages can be done from the admin interface, or each respective cateogry for admins:

Author: ```suggestion
Adding whiteboard messages can be done from the admin interface, or in each respective category for admins:
```

.. image:: /images/whiteboard.png

The whiteboard messages are then shown based on their specified context:

No context specified

    Shown on dashboard (landing page).

Project specified

>     Shown for project, and all its components and translations.

Author: ```suggestion
    Shown for project, including all its components and translations.
```

>     Shown for project, including all its components and translations.

Author: ```suggestion
    Shown within the project, including all its components and translations.
```

Component specified

>     Shown for a component and all its translations.

Author: ```suggestion
    Shown for a given component and all its translations.
```

Language specified

    Shown on the language overview and all translations in that language.

This is how it looks on the language overview page:

.. image:: /images/whiteboard-language.png

>    alt : Image showing a message that reads: "Czech translators rock!" atop the Czech language overview.

Reviewer: ```suggestion
   :alt: Image showing a message that reads: "Czech translators rock!" atop the Czech language overview.
```

And on the project page:

.. image:: /images/whiteboard-project.png

>    alt : Image showing a message that reads: "Translations will be used only if they reach 60%" atop the dashboard view.

Author: This should be discouraged. Can't think of something to replace it right now. (Needs the image too).

>    alt : Image showing a message that reads: "Translations will be used only if they reach 60%" atop the dashboard view.

Reviewer: ```suggestion
    :alt: Image showing a message that reads: "Translations will be used only if they reach 60%" atop the dashboard view.
```
